homeassistant/components/flipr/config_flow.py

```diff
@@ -11,35 +12,37 @@
 from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
 from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
 
-from .const import CONF_FLIPR_ID, DOMAIN
+from .const import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_EMAIL): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
+
 
 class FliprConfigFlow(ConfigFlow, domain=DOMAIN):
-    """Handle a config flow for Flipr."""
+    """Handle a config flow for flipr."""
 
     VERSION = 1
```

Comment:
We should also set the version to 2, and then update the config entry unique id in the migration

Reply:
Done